trail/html/index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE HTML>
+<!--
+	template by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+	<head>
+		<title>trailblzer</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+	</head>
+	<body class="is-preload">
+
+		<!-- Wrapper -->
+			<div id="wrapper" class="fade-in">
+
+				<!-- Intro -->
+					<div id="intro">
+						<h1>trailblazer</h1>
+						<p>Your favorite collection of staggering satellite streaks.</p>

```

Comment:
```suggestion
						<p>An open data repository for astronomical data products affected by satellites</p>

```
Or, if you want to be snarky/add hover text/footnote, "satellite streaks as a service" :)

Reply:
@Landing: Sure, no problem. Check out the new version! I've also altered the color scheme to UW. Let me know it that works or if you wanna change it. @Gallery: This is going to look nothing like the page at the moment once Dino has hacked in the template, so no worries about that at the moment. I just wanted to put some place holder there. Also, there is no content. Once you know what should go on the various pages, we can revisit.